text-ui-test/input.txt

```diff
@@ -0,0 +1,8 @@
+todo borrow book
+deadline return book /by Sunday
+event project meeting /at Mon 2-4pm
+list
+mark 1
+list
+unmark 1
+list
```

Comment:
Some additional test cases you can consider:

1. Invalid comand formats for `deadline`, `event` and `mark`
    - e.g. `deadline /by Sunday`, `event project meeting /by Mon`, `mark a`
2. Tests for `delete` command 
3. What should be the expected output of `list` if there are no tasks yet in the `taskList`
4. Error messages when a invalid command is given
    - Test for the correct error message for commands like `mark a`
5. Tests for unsupported commands (e.g. `hello`)
6. For completeness sake, the expected output of `bye` command 😅